src/app/app.module.ts

```diff
@@ -80,7 +81,8 @@ export function createTranslateLoader(http: HttpClient) {
     HelperModule,
     OpenCaseListSliderModule,
     HomeModule,
-    MyProfileModule
+    MyProfileModule,
+    BreadcrumbModule
```

Comment:
Is that import required? Isn't the `BreadcrumbModule` intended to only be used in the "functional" modules?

Reply:
No, it's not required. Have deleted it.